Package.swift

```diff
@@ -12,6 +12,7 @@ let package = Package(
     ],
     targets: [
         .target(name: "MongoSwift", dependencies: ["mongoc", "bson"]),
-        .testTarget(name: "MongoSwiftTests", dependencies: ["MongoSwift", "Nimble"])
+        .target(name: "AtlasConnectivity", dependencies: ["MongoSwift"]),
+        .testTarget(name: "MongoSwiftTests", dependencies: ["MongoSwift", "Nimble", "mongoc"])
```

Comment:
I think that this has just worked without mongoc there since the driver always gets built first, but I think technically mongoc should be a dependency here as we directly `import libmongoc` in a few test files.